cmd/tailscale/cli/up.go

```diff
@@ -54,6 +54,7 @@ specify any flags, options are reset to their default.
 		upf.StringVar(&upArgs.authKey, "authkey", "", "node authorization key")
 		upf.StringVar(&upArgs.hostname, "hostname", "", "hostname to use instead of the one provided by the OS")
 		upf.BoolVar(&upArgs.enableDERP, "enable-derp", true, "enable the use of DERP servers")
+		upf.BoolVar(&upArgs.corpDNS, "corp-dns", true, "use DNS configuration from the admin panel")
```

Comment:
How about --use-dns ? CorpDNS doesn't make much sense, as you pointed out, and it will make less and less sense over time when we add MagicDNS, domain-only DNS, etc.

I think --use-dns is a fine name for the "globally disable all DNS stuff" switch.

Reply:
Yep, that makes great sense. Changed.